openmetadata-ui/src/main/resources/ui/src/components/common/CronEditor/CronEditor.tsx

```diff
@@ -564,10 +599,10 @@ const CronEditor = (props) => {
             <label htmlFor="ingestionType">Every:</label>
             <select
               className="tw-form-inputs tw-px-3 tw-py-1"
+              data-testid="ingestion-type"
```

Comment:
`id` should be something related to cron and not ingestion
It's minor but something to keep in mind while assigning it

Reply:
Thanks, @chirag-madlani, I will change that.